templates/terms.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<p>Terms of Service.<br>
```

Comment:
Don't forget to close this <p> tag :)

Reply:
Hah, that's funny. I had put don't forget to close this `<p>` tag... and it got read in.